src/conduits/java/com/enderio/conduits/common/blockentity/connection/StaticConnectionStates.java

```diff
@@ -0,0 +1,21 @@
+package com.enderio.conduits.common.blockentity.connection;
+
+/**
+ * This class is for all static ConnectionStates without being connected to a block. For this use {@link DynamicConnectionState} <br>
```

Comment:
Which state would hold "can connected to block but disabled"? Disconnected? It's unclear to me (but that can be just me) if "without being connected to a block" means not being near one, or not being allowed to.

Reply:
Connected to a block means having IO/CapabilityAttachment of some sorts. Like connecting to a chest or barrel or machine. I'm not sure which state you are desrcibing tbh

Reply:
If I manually decide not to connect to a chest. Which state is that?

Reply:
that is disabled. if you remove an existing connection by disabling both input and output (if we are talking about itemconduits), other conduits can do the same but oc conduits have an enabled field, the connection was stopped and wouldn't reconnect to anything without a wrench saying that you can connect again

Reply:
Ok, seems fine. I was just wondering, figured it would be that but wanted to be sure.